src/module.c

```diff
@@ -1126,10 +1126,10 @@ int RM_ReplyWithLongLong(RedisModuleCtx *ctx, long long ll) {
 int replyWithStatus(RedisModuleCtx *ctx, const char *msg, char *prefix) {
     client *c = moduleGetReplyClient(ctx);
     if (c == NULL) return REDISMODULE_OK;
-    sds strmsg = sdsnewlen(prefix,1);
-    strmsg = sdscat(strmsg,msg);
-    strmsg = sdscatlen(strmsg,"\r\n",2);
-    addReplySds(c,strmsg);
+    const size_t len = strlen(msg);
+    addReplyProto(c,"-",1);
```

Comment:
This should be `prefix` I think not hardcoded `-`.

Reply:
you're totally right @antirez. Sorry for that =)